### PR TITLE
ci: migrate to GitHub Actions instead

### PR DESCRIPTION
**Pull request by JounQin**
@wooorm Here we go.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,8 +10,7 @@
   </a>
 </p>
 
-[![Travis](https://img.shields.io/travis/com/mdx-js/eslint-mdx.svg)](https://travis-ci.com/mdx-js/eslint-mdx)
-[![Codacy Grade](https://img.shields.io/codacy/grade/4ea8225261c04837995a858676caae4b)](https://www.codacy.com/app/JounQin/eslint-mdx)
```
Comment (JounQin) [inline README.md:14]: @wooorm Can you help to enable https://app.codacy.com/manual/mdx-js/eslint-mdx for this repo only? I think codacy is a good service to display the project code grade.

Comment (wooorm) [reply README.md:14]: I would really prefer to keep such tools to a minimum.
Each new service provides a new vector to be hacked.
Recently codecov had an incident: https://about.codecov.io/security-update/.
The *less* services you use, the more secure things are.

Can you do without it?

Comment (JounQin) [reply README.md:14]: OK, not a required one.

Comment (JounQin) [reply README.md:14]: @wooorm Good to approve then?

Comment (wooorm) [reply README.md:14]: Yep, looks good otherwise!!

**Review by wooorm — APPROVED** [review]

**Pull request merged**